package.json

```diff
@@ -11,32 +11,47 @@
     "write-good": "bin/write-good.js",
     "writegood": "bin/write-good.js"
   },
+  "engines": {
+    "node": ">=6",
+    "npm": ">=5"
+  },
   "directories": {
     "test": "test"
   },
   "scripts": {
-    "test": "./node_modules/.bin/jasmine-node test"
+    "test": "npm run lint && ./node_modules/.bin/jasmine-node test",
+    "lint": "eslint ./ --fix"
   },
   "keywords": [
     "english",
     "writing",
     "prose",
-    "words"
+    "words",
+    "grammar"
   ],
   "author": "Brian Ford",
+  "contributors": [
+    { "name": "Richard Littauer", "email": "richard@maintainer.io", "url": "https://burntfen.com" },
```

Comment:
`npm i` will naturally format these as pretty, making a changed file. Can you run it once and commit the changes?

Reply:
👍